### DISCUSSION TITLE
Fido2 Passwordless Login

### DISCUSSION THREAD

**Discussion by quexten** [dani-garcia/vaultwarden#4116] [Ideas]
Upstream recently merged passwordless fido2 login in the backend and frontend (just on master for now). Creating this discussion to see what's required for the feature to be implemented.

Client:
https://github.com/bitwarden/clients/pull/6798

Server:
https://github.com/bitwarden/server/pull/3467
https://github.com/bitwarden/server/pull/3361
https://github.com/bitwarden/server/pull/2915
https://github.com/bitwarden/server/pull/2885

As far as I can tell it should mostly be implementing a few new API endpoints and creating the database queries.

**Comment by metal0** [comment]
also another minor client PR:
[#6861](https://github.com/bitwarden/clients/pull/6861)

**Comment by ReLoia** [comment]
any update on this discussion? Is there any stated reason on why we can't login without a password?

**Reply by BlackDex** [reply]
Because t that feature isn't part of Vaultwarden yet. It's that simpel.

There is a PR open for it. #5929
